styles/components/SidebarItem.module.scss

```diff
@@ -0,0 +1,30 @@
+@import "../variables";
+
+.sidebarItem {
+    display: flex;
+    flex-direction: row;
+    width: 90%;
+    padding: 5px;
+    border-radius: 10px;
+    font-size: 25px;
+    transition: 400ms;
```

Comment:
```suggestion
```

If this class is related to hover effect on sidebar items then remove it. Incase it is not, find it and remove that transition property.

![image](https://github.com/PEC-CSS/website/assets/75680424/61469206-6c19-4bd7-904d-5dccfd7c5276)

Reply:
Why should we remove this though? 🤔